### PR TITLE
(feat) O3-4026: Add an icon for the vitals and biometrics link to the left panel

### PR DESCRIPTION
**Pull request by Munyua123**
## Requirements

- [x] This PR has a title that briefly describes the work done including the ticket number. If there is a ticket, make sure your PR title includes a [conventional commit](https://o3-docs.openmrs.org/docs/frontend-modules/contributing.en-US#contributing-guidelines) label. See existing PR titles for inspiration.
- [ ] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide) and [design documentation](https://om.rs/o3ui).
- [ ] My work includes tests or is validated by existing tests.

## Summary

I have added a function that allows someone to have an icon before the nav menu title

## Screenshots

image
image

### CODE REVIEW

**Review by CynthiaKamau — APPROVED** [review]
Thanks @Munyua123 for this, kindly add a video or image so that guys can easily see the new changes

**Comment by Munyua123** [comment]
@CynthiaKamau I have added the screenshots

File: packages/esm-patient-common-lib/src/dashboards/DashboardExtension.tsx [hunk 1]
```diff
@@ -22,13 +23,18 @@ export const DashboardExtension = ({
   const location = useLocation();
   const navLink = useMemo(() => decodeURIComponent(last(location.pathname.split('/'))), [location.pathname]);
 
+  const renderIcon = title === 'Vitals & Biometrics' ? <Activity style={{ marginRight: '8px' }} /> : null;
+
   return (
-    <div key={path}>
+    <div key={path} style={{ display: 'flex', alignItems: 'center' }}>
```
Comment (denniskigen) [inline packages/esm-patient-common-lib/src/dashboards/DashboardExtension.tsx:29]: Can we move these styles to an `.scss` file?

File: packages/esm-patient-common-lib/src/dashboards/DashboardExtension.tsx [hunk 2]
```diff
@@ -22,13 +23,18 @@ export const DashboardExtension = ({
...
       <ConfigurableLink
         className={classNames('cds--side-nav__link', { 'active-left-nav-link': path === navLink })}
         to={`${basePath}/${encodeURIComponent(path)}`}
       >
-        {t(title)}
+        <span style={{ display: 'flex', alignItems: 'center' }}>
```
Comment (denniskigen) [inline packages/esm-patient-common-lib/src/dashboards/DashboardExtension.tsx:34]: Ditto

File: packages/esm-patient-common-lib/src/dashboards/DashboardExtension.tsx [hunk 3]
```diff
@@ -22,13 +23,18 @@ export const DashboardExtension = ({
   const location = useLocation();
   const navLink = useMemo(() => decodeURIComponent(last(location.pathname.split('/'))), [location.pathname]);
 
+  const renderIcon = title === 'Vitals & Biometrics' ? <Activity style={{ marginRight: '8px' }} /> : null;
```
Comment (denniskigen) [inline packages/esm-patient-common-lib/src/dashboards/DashboardExtension.tsx:26]: Can we use [ActivityIcon](https://github.com/openmrs/openmrs-esm-core/blob/69859408fa1b7bf773a89002b852cab79a1ae309/packages/framework/esm-styleguide/src/icons/icons.tsx#L14) here instead?

**Review by denniskigen — CHANGES_REQUESTED** [review]
Thanks, @Munyua123. I've left some comments.

**Comment by Munyua123** [comment]
@denniskigen All the comments above have been noted and iam making the changes

**Pull request closed without merging**